crates/web5/src/dids/document.rs

```diff
@@ -10,8 +10,8 @@ pub struct Document {
     pub controller: Option<Vec<String>>,
     #[serde(rename = "alsoKnownAs", skip_serializing_if = "Option::is_none")]
     pub also_known_as: Option<Vec<String>>,
-    #[serde(rename = "verificationMethod")]
-    pub verification_method: Vec<VerificationMethod>,
+    #[serde(rename = "verificationMethod", skip_serializing_if = "Option::is_none")]
+    pub verification_method: Option<Vec<VerificationMethod>>,
```

Comment:
What was the motivating factor for moving this property to optional?

We currently require at least 1 verification method via [our web5-spec](https://github.com/TBD54566975/web5-spec/blob/main/spec/did.md#did-document-data-model), albeit this is not to be considered entirely orthodox and open to changing.

Reply:
Oops, I can see this has already been discussed in a prior comment. Apologies. Disregard this comment.